<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deflectpoint</class>
 <widget class="QMainWindow" name="deflectpoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>挠度监测点信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(38, 183, 248)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>471</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>监测点名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>监测点编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>是否已设置传感器</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="savepushButton">
       <property name="text">
        <string>保存信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delepushButton">
       <property name="text">
        <string>删除监测点信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>281</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>                             监测点名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="namelineEdit">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numlabel">
       <property name="text">
        <string>                              监测点编号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numlineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addpushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>增加监测点信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="returnpushButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setsensorpushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="text">
     <string>设置传感器</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pointnumlineEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deflectlinepushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>显示测点曲线图</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
